src/main/scala/io/iohk/ethereum/blockchain/sync/SyncController.scala

```diff
@@ -103,23 +103,26 @@ class SyncController(
 }
 
 object SyncController {
-  def props(peerManager: ActorRef,
-            appStateStorage: AppStateStorage,
+  def props(appStateStorage: AppStateStorage,
             blockchain: Blockchain,
             blockchainStorages: BlockchainStorages,
             syncStateStorage: FastSyncStateStorage,
             ledger: Ledger,
             validators: Validators,
-            peerMessageBus: ActorRef):
-  Props = Props(new SyncController(peerManager, appStateStorage, blockchain, blockchainStorages,
-    syncStateStorage, ledger, validators, peerMessageBus))
+            actors: DependencyActors):
+  Props = Props(new SyncController(appStateStorage, blockchain, blockchainStorages,
+    syncStateStorage, ledger, validators, actors))
+
+  case class DependencyActors(peerManager: ActorRef, peerMessageBus: ActorRef, pendingTransactionsManager: ActorRef)
```

Comment:
What scalastyle error exactly? 
This is used only for sync controller, what if we want to pass a different set of actors elsewhere? Will we create a new class for each such case?
Also the name `DependencyActors` is kind of like calling a variable `aVariable`. But it's hard to find a good name since they are unrelated actors from different packages - and this is probably the cue that they shouldn't be grouped.